po/hi.po

```diff
@@ -690,7 +690,7 @@ msgstr "अमान्य --require-version तर्क: %s"
 #: app/flatpak-builtins-build-finish.c:533
 #, c-format
 msgid "Too few elements in --extra-data argument %s"
-msgstr "अतिरिक्त डेटा तर्क में बहुत कम तत्व %s"
+msgstr "--extra-data तर्क %s . में बहुत कम तत्व %s"
```

Comment:
I think the issue is you have `%s`  twice here whereas the untranslated string has it once.

Reply:
Thanks! This is where the problem lies.
This is currently the last patch for the Hindi translation.